### PR TITLE
Refactor CoordsXYZ

### PR DESCRIPTION
**Pull request by 0Cufox0**
Just implemented CoordsXYZ into different classes

### CODE REVIEW

**Comment by janisozaur** [comment]
`Replay/ReplayTests.RunReplay/issue8203` test is failing

File: CMakeFiles/cmake.check_cache [hunk 1]
```diff
@@ -0,0 +1 @@
+# This file is generated by cmake for dependency checking of the CMakeCache.txt file
```
Comment (janisozaur) [inline CMakeFiles/cmake.check_cache]: This should not be part of the commit

File: src/openrct2-ui/windows/Main.cpp [hunk 2]
```diff
@@ -61,6 +61,7 @@ static rct_window_event_list window_main_events = {
  */
 rct_window* window_main_open()
 {
+
```
Comment (janisozaur) [inline src/openrct2-ui/windows/Main.cpp]: Remove

File: src/openrct2/core/Console.cpp [hunk 3]
```diff
@@ -18,6 +18,7 @@ namespace Console
 {
     void Write(char c)
     {
+
```
Comment (janisozaur) [inline src/openrct2/core/Console.cpp]: Remove

File: CMakeFiles/cmake.check_cache (hunk 1 above)
Comment (0Cufox0) [reply CMakeFiles/cmake.check_cache]: > `Replay/ReplayTests.RunReplay/issue8203` test is failing

Just removed all the whiteSpaces, any idea on how I'd fix this?

**Comment by 0Cufox0** [comment]
Oh sorry I didnt know, I'll look into it again and try my best!

**Comment by Gymnasiast** [comment]
If you need more explanation about the two, you can ask in the Gitter channel.

Basically:
Tile Coordinates increment X and Y by 1 per tile, and Z by 1 per 0.75 metres (2.5 ft).
(Big) Coordinates increment X and Y by 32 per tile, and Z by 8 per 0.75 metres.

You often see multiplication or division by 32 and 8 to convert between the two (also expressed as shifts by 5 and 3, respectively).

**Comment by 0Cufox0** [comment]
> If you need more explanation about the two, you can ask in the Gitter channel.
> 
> Basically:
> Tile Coordinates increment X and Y by 1 per tile, and Z by 1 per 0.75 metres (2.5 ft).
> (Big) Coordinates increment X and Y by 32 per tile, and Z by 8 per 0.75 metres.
> 
> You often see multiplication or division by 32 and 8 to convert between the two (also expressed as shifts by 5 and 3, respectively).

ohhhhh that cleared a lot for me, I think I got a firmer grip on what I should do, thank you!

**Comment by duncanspumpkin** [comment]
Don't close and open pr's. We lose track of the discussion doing that.
